sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
   - `length` is the number of characters in the text of these models
   - `offset` is the offset of the text from the start of the document
   
+**New features**
```

Comment:
Shouldn't this be 
`### New Features`?

Reply:
we already follow this pattern in all released changelog. I saw Storage does the same.